### PR TITLE
Implement methods

### PR DESCRIPTION
**Pull request by V1talie**

### CODE REVIEW

File: src/main/java/com/endava/internship/service/UserServiceImpl.java [hunk 1]
```diff
@@ -2,68 +2,103 @@
 
 import com.endava.internship.domain.Privilege;
 import com.endava.internship.domain.User;
-import com.endava.internship.service.UserService;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```
Comment (GAlexandru-be) [inline src/main/java/com/endava/internship/service/UserServiceImpl.java]: can you optimize this?

**Comment by GAlexandru-be** [comment]
good job!

**Review by GAlexandru-be — APPROVED** [review]

File: src/main/java/com/endava/internship/service/UserServiceImpl.java [hunk 2]
```diff
@@ -2,68 +2,103 @@
...
-        throw new UnsupportedOperationException("Not implemented");
+        return users.stream()
+                .map(mapFun)
+                .collect(Collectors.joining(delimiter));
     }
 
     @Override
-    public Map<Privilege, List<User>> groupByPrivileges(List<User> users) {
-        throw new UnsupportedOperationException("Not implemented");
+    public Map<Privilege, List<User>> groupByPrivileges(List<User> users) {//????????????????????????????
```
Comment (GAlexandru-be) [inline src/main/java/com/endava/internship/service/UserServiceImpl.java]: remove comments

File: src/main/java/com/endava/internship/service/UserServiceImpl.java (hunk 1 above)
Comment (V1talie) [reply src/main/java/com/endava/internship/service/UserServiceImpl.java]: Done

File: src/main/java/com/endava/internship/service/UserServiceImpl.java (hunk 2 above)
Comment (V1talie) [reply src/main/java/com/endava/internship/service/UserServiceImpl.java]: Done

**Pull request merged**